src/components/rangeslider/draw.js

```diff
@@ -395,12 +396,22 @@ function drawBg(rangeSlider, gd, axisOpts, opts) {
     var offsetShift = -opts._offsetShift;
     var lw = Drawing.crispRound(gd, opts.borderwidth);
 
+    var fillColor = tinycolor(opts.bgcolor);
+    var fillAlpha = fillColor.getAlpha();
+    var fillRGB = Color.tinyRGB(fillColor);
+
+    var strokeColor = tinycolor(opts.bordercolor);
+    var strokeAlpha = strokeColor.getAlpha();
+    var strokeRGB = Color.tinyRGB(strokeColor);
+
     bg.attr({
         width: opts._width + borderCorrect,
         height: opts._height + borderCorrect,
         transform: strTranslate(offsetShift, offsetShift),
-        fill: opts.bgcolor,
-        stroke: opts.bordercolor,
+        fill: fillRGB,
+        'fill-opacity': fillAlpha,
+        stroke: strokeRGB,
+        'stroke-opacity': strokeAlpha,
```

Comment:
🌴 better to reuse `Color.stroke` and `Color.fill` here and in [selections/select.js](https://github.com/plotly/plotly.js/pull/6323/files#diff-001f0152e25c2d121012cb729df68fb3786434bc98955733328715978a11f60a) - and that would also help address the presumably unintentional use of `attr` here whereas elsewhere we use `style` for these.

To be specific, you can either:
```js
bg.attr({...})
.call(Color.stroke, opts.bordercolor)
.call(Color.fill, opts.bgcolor);
```
or:
```js
bg.attr({...});
Color.stroke(bg, opts.bordercolor);
Color.fill(bg, opts.bgcolor);
```